mobile/src/main/java/ch/epfl/sdp/mobile/state/StatefulProfileScreen.kt

```diff
@@ -36,8 +34,14 @@ fun StatefulProfileScreen(
     modifier: Modifier = Modifier,
 ) {
   val socialFacade = LocalSocialFacade.current
+  val chessFacade = LocalChessFacade.current
   val profile by remember(socialFacade, uid) { socialFacade.profile(uid) }.collectAsState(null)
-  val state = profile?.let { FetchedUserProfileScreenState(it) }
+  val matches =
+      remember(chessFacade, profile) { profile?.let { chessMatches(it, chessFacade) } }
+          ?.collectAsState(emptyList())
+          ?.value
+          ?: emptyList()
+  val state = profile?.let { FetchedUserProfileScreenState(it, matches) }
   if (state != null) {
```

Comment:
This is some legacy code, but having a `null` `state` means something went terribly wrong, and we should get rid of it.

Reply:
Fixed in 9337db8ae78de9150eed15e056e78fb9165dd16a